lib/req/utils.ex

```diff
@@ -504,6 +509,12 @@ defmodule Req.Utils do
             end)
 
           {stream, size, options}
+
+        enum ->
+          Enumerable.impl_for!(enum)
+          size = Keyword.get(options, :content_length, 0)
```

Comment:
@knightpp oh, I don't think `0` is a good default:

```
iex> Req.post!("https://httpbin.org/anything", form_multipart: [file: {Stream.cycle(["abc"]) |> Stream.take(3), filename: "a.txt"}]).body["files"]
%{}
iex> Req.post!("https://httpbin.org/anything", form_multipart: [file: {Stream.cycle(["abc"]) |> Stream.take(3), filename: "a.txt", content_length: 9}]).body["files"]
%{"file" => "abcabcabc"}
```

should I change this to `size = Keyword.fetch!(options, :content_length)`?